scripts/skills/effects/rf_sapling_harvest_effect.nut

```diff
@@ -0,0 +1,49 @@
+this.rf_sapling_harvest_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		DamageThreshold = 0.1,
+		SpawnedUnit = "scripts/entity/tactical/enemies/schrat_small"
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_sapling_harvest";
+		this.m.Name = "Sapling Harvest";
+		this.m.Icon = "skills/terrain_icon_06.png";
+		this.m.SoundOnUse = [];
+		this.m.Type = this.Const.SkillType.StatusEffect;
+		this.m.Order = this.Const.SkillOrder.Last;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = false;
+	}
+
+	function getDescription()
+	{
+		"Whenever this character receives " + ::MSU.Text.colorGreen((this.m.DamageThreshold * 100.0) + "%") + " or more hitpoint damage from a single hit, spawn a Sapling on an adjacent tile, if possible.";
```

Comment:
replace `or more hit point damage` with `or more damage to Hitpoints` to be more in line with vanilla style.